Makefile

```diff
@@ -323,29 +323,26 @@ generate-hubble-api: api/v1/flow/flow.proto api/v1/peer/peer.proto api/v1/observ
 
 define generate_k8s_api
```

Comment:
Do we need these changes? I know that kube_codegen.sh was introduced but if the older script it's still available let's use it. Changeing to the `kube_codegen.sh` will require more changes IIRC.

Reply:
The old script is available, but didn't do anything right ? :thinking: 

https://github.com/kubernetes/code-generator/blob/v0.30.0-rc.1/generate-internal-groups.sh

> echo "ERROR: $(basename "$0") has been removed."
echo "ERROR: Please use k8s.io/code-generator/kube_codegen.sh instead."

Reply:
Yes it was removed and we had to change the scripts to use `kube_codegen.sh`.